x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/util/filter_runtime_mappings.ts

```diff
@@ -86,6 +95,22 @@ function findFieldsInAgg(obj: Record<string, any>) {
   return fields;
 }
 
+function findFieldsInQuery(obj: Record<string, any>) {
```

Comment:
If I'm not mistaken we have some `Query` type provided by the data plugin, but otherwise `Record<string, object  | null>` would be better